opennms-doc/guide-admin/src/asciidoc/text/operation/daemon-config-files/pollerd.adoc

```diff
@@ -0,0 +1,23 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../../images
+
+[[ga-opennms-operation-daemon-config-files-pollerd]]
+==== Pollerd
+
+[options="header, autowidth"]
+|===
+| Internal Daemon Name | Reload Event
+| _Pollerd_            | `uei.opennms.org/internal/reloadDaemonConfig -p 'daemonName Pollerd'`
+|===
+
+.Pollerd configuration file overview
+[options="header, autowidth"]
+|===
+| File                        | Restart Required | Reload Event | Description
+| `poller-configuration.xml`  | yes              | yes          | Restart is required in case new monitors are created or removed.
+                                                                  Reload Event loads changed configuration parameters of existing monitors.
+| `response-graph.properties` | no               | no           | Graph definition for response time graphs from monitors
+| `poll-outages.xml`          | ?                | ?            | ?
```

Comment:
- poller-configuration.xml

It is the main configuration file for Pollerd, and it can be reloaded using reloadDaemonConfig.

- poll-outages.xml

It is related with the scheduled outages feature. I'm not sure if it is reloaded automatically or not.

- response-graph.properties

It contains the graph templates for the metrics collected by Pollerd, but the file itself is not managed by Pollerd. Any graph related file is reloaded automatically, every time it is changed.